src/shared/ml-thirdparty-client/models/core/transactions.ts

```diff
@@ -35,3 +54,137 @@ export enum Currency {
   SGD = 'SGD',
   USD = 'USD',
 }
+
+/**
+ * Data model for the complex type GeoCode. 
+ * Indicates the geographic location of a transaction initiation.
+ */
+export interface GeoCode {
+  /**
+   * Latitude of the Party.
+   */
+  latitude: string
+
+  /**
+   * Longitude of the Party.
+   */
+  longitude: string
+}
+
+/**
+ * Data model for the complex type Money.
+ */
+export interface Money {
+  /**
+   * Amount of money.
+   */
+  amount: string
+
+  /**
+   * Currency of the amount.
+   */
+  currency: Currency
+}
+
+/**
+ * Quote of a transaction.
+ */
+export interface Quote {
+  /**
+   * The amount of money that the Payee FSP should receive.
+   */
+  transferAmount: Money
+
+  /**
+   * The amount of Money that the Payee should receive in the end-to-end transaction. 
+   * Optional as the Payee FSP might not want to disclose any optional Payee fees.
+   */
+  payeeReceiveAmount?: Money
+
+  /**
+   * Payee FSP’s part of the transaction fee.
+   */
+  payeeFspFee?: Money
+
+  /**
+   * Transaction commission from the Payee FSP.
+   */
+  payeeFspComission?: Money
+
+  /**
+   * Date and time until when the quotation is valid and can be honored when used 
+   * in the subsequent transaction.
+   */
+  expiration: string
+
+  /**
+   * Longitude and Latitude of the Payee. Can be used to detect fraud.
+   */
+  geoCode?: GeoCode
+
+  /**
+   * Information for recipient (transport layer information).
+   */
+  ilpPacket: string
+
+  /**
+   * Condition that must be attached to a transfer by the Payer.
+   */
+  condition: string
+
+  /**
+   * Optional extension, specific to deployment.
+   */
+  extensionList?: ExtensionList
+}
+
+/**
+ * Data model for the complex type Refund.
+ */
+export interface Refund {
+  /**
+   * Reference to the original transaction ID that is requested to be refunded.
+   */
+  originalTransactionId: string
+
+  /**
+   * Free text indicating the reason for the refund.
+   */
+  refundReason?: string
+}
+
+/**
+ * 
+ */
+export interface TransactionType {
+  /**
+   * Deposit, withdrawal, refund, …
+   */
+  scenario: string
+
+  /**
+   * Possible sub-scenario, defined locally within the scheme.
+   */
+  subScenario?: string
+
+  /**
+   * Who is initiating the transaction - Payer or Payee.
+   */
+  initiator: string
+
+  /**
+   * Consumer, agent, business, …
+   */
+  intiiatorType: string
```

Comment:
typo

Reply:
Oops... sorry didn't notice this one, fixed :)